transactions/transactions_service.go

```diff
@@ -1460,3 +1478,31 @@ func (svc *transactionsService) markPaymentFailed(tx *gorm.DB, dbTransaction *db
 	})
 	return nil
 }
+
+// NOTE: we wouldn't need this if we resubscribe to pending swaps
```

Comment:
can we do this instead? it feels wrong to do this from the transactions service. I don't think it should know about swaps at all